sbv 5.15

Ping @acfoltzer, cryptol-2.4.0 fails with: 

```
[75 of 78] Compiling Cryptol.Symbolic ( src/Cryptol/Symbolic.hs, dist/build/Cryptol/Symbolic.o )

src/Cryptol/Symbolic.hs:129:37: error:
    • Couldn't match expected type ‘sbv-5.15:Data.SBV.Utils.TDiff.Timing’
                  with actual type ‘Bool’
    • In the ‘timing’ field of a record
      In the expression:
        p {SBV.timing = pcVerbose, SBV.verbose = pcVerbose}
      In the expression:
        [p {SBV.timing = pcVerbose, SBV.verbose = pcVerbose} |
           p <- provers]
```

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -1282,7 +1282,7 @@ packages:
         - language-thrift
 
     "Adam C. Foltzer <acfoltzer@galois.com> @acfoltzer":
-        - cryptol
+        # - cryptol # bounds: sbv
         - gitrev
         - persistent-refs
 
@@ -3156,9 +3156,6 @@ packages:
 
         # https://github.com/fpco/stackage/issues/2127
         - leapseconds-announced < 2017.0.0.1
-
-        # https://github.com/fpco/stackage/issues/2251
-        - sbv < 5.15
 
         # https://github.com/fpco/stackage/issues/2317
         - skylighting < 0.2

